core/application/impl/app_configuration_impl.cpp

```diff
@@ -175,11 +176,12 @@ namespace {
 
   static constexpr std::array<std::string_view,
                               1 + KAGOME_WASM_COMPILER_WASM_EDGE>
-      interpreters{
+      interpreters {
 #if KAGOME_WASM_COMPILER_WASM_EDGE == 1
-          "WasmEdge",
+    "WasmEdge",
 #endif
-          "Binaryen"};
+        "Binaryen"
```

Comment:
Since line 181 has been changed, please do the same indent for both 181 and 183.